rsocket-core/src/main/java/io/rsocket/RSocketRequester.java

```diff
@@ -110,7 +110,7 @@
           new ClientKeepAliveSupport(allocator, keepAliveTickPeriod, keepAliveAckTimeout);
       this.keepAliveFramesAcceptor =
           keepAliveHandler.start(
-              keepAliveSupport, sendProcessor::onNext, this::tryTerminateOnKeepAlive);
+              keepAliveSupport, sendProcessor::onNextPrioritized, this::tryTerminateOnKeepAlive);
```

Comment:
We should tighten this up in the spec, whether it's stream 0 prioritised or KEEPALIVE and LEASE only as you have implemented.  I'll generally defer to @robertroeser for this, but put my thoughts on the issue.

Reply:
Definitely. I have already created an issue on that!